server/router/applicant-router.js

```diff
@@ -0,0 +1,7 @@
+const applicantRouter = require('express').Router();
```

Comment:
@ahmadysalah I think the most common convention with imports is to separate them from "using" that required module or calling functions on it.  So for example, this would follow the typical convention better:
```
const express = require('express');
const applicantRouter = express.Router();
```